apps/website/src/core/util/tailwind-variants.util.ts

```diff
@@ -0,0 +1,14 @@
+import { tv as tvBase } from 'tailwind-variants';
+import type { TV } from 'tailwind-variants';
+import { tailwindMergeConfig } from './tw-merge.util';
+
+export const tv: TV = (options, config) => {
+  return tvBase(options, {
+    ...config,
+    twMerge: true,
+    // Tailwind Variantsの型定義が依存パッケージの`tailwind-merge`にそぐわないものなので、型を強制的に上書きする
+    // このIssueが解決されたら、型の上書きを削除する
```

Comment:
一応 todo tree で捕捉できるように 🙏


```suggestion
    // HACK: Tailwind Variantsの型定義が依存パッケージの`tailwind-merge`にそぐわないものなので、型を強制的に上書きする
    // TODO: このIssueが解決されたら、型の上書きを削除する
```